apps/ybal/components/HarvestsListRow.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import IconLinkOut from '@yearn-finance/web-lib/icons/IconLinkOut';
+import {toAddress, truncateHex} from '@yearn-finance/web-lib/utils/address';
+import {STYBAL_TOKEN_ADDRESS} from '@yearn-finance/web-lib/utils/constants';
+import {formatBN, formatToNormalizedValue} from '@yearn-finance/web-lib/utils/format.bigNumber';
+import {formatAmount, formatUSD} from '@yearn-finance/web-lib/utils/format.number';
+import {formatDate} from '@yearn-finance/web-lib/utils/format.time';
+import {ImageWithFallback} from '@common/components/ImageWithFallback';
+
+import type {ReactElement} from 'react';
+import type {TYDaemonHarvests} from '@common/types/yearn';
+
+function	HarvestListRow({harvest}: {harvest: TYDaemonHarvests}): ReactElement {
+	return (
+		<div className={'yearn--table-wrapper'}>
+			<div className={'yearn--table-token-section'}>
+				<div className={'yearn--table-token-section-item'}>
+					<div className={'yearn--table-token-section-item-image'}>
+						<ImageWithFallback
+							alt={toAddress(harvest.vaultAddress) === STYBAL_TOKEN_ADDRESS ? 'st-yBal' : 'lp-yBal'}
+							width={40}
+							height={40}
+							quality={90}
+							src={`${process.env.BASE_YEARN_ASSETS_URI}/1/${toAddress(harvest.vaultAddress)}/logo-128.png`}
+							loading={'eager'} />
+					</div>
+					<p>
+						{toAddress(harvest.vaultAddress) === STYBAL_TOKEN_ADDRESS ? 'st-yBal' : 'lp-yBal'}
+					</p>
+				</div>
+			</div>
+
+			<div className={'yearn--table-data-section md:grid-cols-9'}>
+				<div className={'yearn--table-data-section-item md:col-span-1'} datatype={'number'}>
+					<p className={'yearn--table-data-section-item-label'}>{'Gain'}</p>
+					<b className={'yearn--table-data-section-item-value'}>
+						{formatAmount(formatToNormalizedValue(formatBN(harvest.profit).sub(formatBN(harvest.loss)), 18))}
```

Comment:
We're formatting quite a bit here, I was thinking we could try and have zod parse the values to BigNumber (or other more preferable formats than what we get over the wire).

Then, we have a `toNormalizedAmount` in the web-lib that seems to be doing what we want here, e.g.
```typescript
export const	toNormalizedAmount = (v: BigNumberish, d?: number): string => (
	formatAmount(toNormalizedValue(v, d ?? 18), 6, 6)
);
```

It also defaults to 18 decimal places, so we can omit that too. Although, shall we double check that the we are getting 18 decimal places? I remember recently there was a bug where we assumed we were getting 18 and it was 6 for that particular case.

So with all the above, we could simply
```typescript
toNormalizedAmount(harvest.profit.sub(harvest.loss))
```